api/tests/core/search/test_commands_indexation.py

```diff
@@ -125,3 +127,16 @@ def test_partially_index_venues(app):
     # fmt: on
 
     assert set(search_testing.search_store["venues"].keys()) == expected_to_be_reindexed
+
+    search_testing.reset_search_store()
+    # --- delete until here when legacy command is removed
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🤓 nitpick:** ‏`up to here` me parait plus compréhensible, mais bon very low logaf
Avec le message au dessus c'est assez clair